source/LifecycleMethod.cpp

```diff
@@ -248,7 +248,55 @@ bool LifecycleMethod::validate(
   } else {
     // TODO:handle graph
     const auto& graph = std::get<LifeCycleMethodGraph>(body_);
-    static_cast<void>(graph); // hide unused variable warning.
+    if (graph.get_node("entry") == nullptr) {
+      ERROR(
+          1,
+          "Entry point entry is not a valid node in the lifecycle graph.");
+      return false;
```

Comment:
In `LifecycleMethodCall::validate()` we actually throw an exception `LifecycleMethodValidationError` on errors. Could we do the same here?